doc/source/user_guide/solver_settings/index.rst

```diff
@@ -100,23 +100,23 @@ as a dictionary for ``Group`` and ``NamedObject`` types or as a list for ``ListO
 .. code-block::
 
   >>> import ansys.fluent.core as pyfluent
-  >>> viscous = pyfluent.Viscous(settings_source=solver)
+  >>> viscous = pyfluent.solver.Viscous(settings_source=solver)
```

Comment:
@mkundu1, any reason we chose "solver" over "settings"?. I was thinking "settings" to be more apt here as in the verbose form we are using "settings".

Reply:
@prmukherj In the context of the `ansys.fluent.core` namespace, the `settings` namespace does not carry sufficient meaning. `settings` there could just as easily refer to miscellaneous library configuration settings.

Reply:
ok, got it. Thanks @seanpearsonuk